docs/sanic/logging.md

```diff
@@ -14,7 +14,7 @@ from sanic.config import LOGGING
 # The default logging handlers are ['accessStream', 'errorStream']
 # but we change it to use other handlers here for demo purpose
 LOGGING['loggers']['network']['handlers'] = [
-    'accessTimedRotatingFile', 'errorTimedRotationgFile']
+    'accessSysLog', 'errorSysLog']
```

Comment:
Needs to update https://github.com/channelcat/sanic/pull/724/files#diff-7e3514b34ffada2985c8db3b21bf4794R29 too.